Pod/Classes/VOKCoreDataManager.m

```diff
@@ -93,7 +95,7 @@ - (NSManagedObjectContext *)tempManagedObjectContext
         return nil;
     }
 
-    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSConfinementConcurrencyType];
+    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
```

Comment:
I suppose the correct course is to follow Apple's lead and start using the context's queues rather than managing our own.